mbf_utility/include/mbf_utility/navigation_utility.h

```diff
@@ -107,6 +109,20 @@ bool getRobotPose(const TF &tf,
                   const std::string &global_frame,
                   const ros::Duration &timeout,
                   geometry_msgs::PoseStamped &robot_pose);
+
+/**
+ * @brief subscribe odom_topic callback
+ * @param sub_odometry subscribe data from wheel odometry
+ */           
+void actualVelCallback(const nav_msgs::Odometry::ConstPtr& sub_odometry);
+
+/**
+ * @brief update the robot actual velocity.
+ * @param tf_listener TransformListener.
+ * @return true, if succeeded, false otherwise.
+ */
+void getRobotActualVelocity(geometry_msgs::TwistStamped &robot_velocity);
```

Comment:
Just as note. A nice way is using `std::optional` in case of C++17 or boost::optional.

Do you still stick with 
```
void getRobotActualVelocity(geometry_msgs::TwistStamped &robot_velocity);

// instead of
std::optional<geometry_msgs::TwistStamped> getOdometryTwistStamped();
```